packages/story-editor/src/components/panels/design/textStyle/karma/textStyle.other.karma.js

```diff
@@ -290,8 +290,9 @@ describe('Text Style Panel', () => {
       const DEFAULT_CUSTOM_FONTS = 2;
       // Timeout used for submitting / search update + 250ms (250 + 250).
       const TIMEOUT = 500;
-      const openFontPicker = async () => {
-        const input = fixture.screen.getByLabelText('Font family');
+      const openFontPicker = async (i = 0) => {
```

Comment:
Maybe add it to the `TextStyle` class in the `karma` folder instead? To make it available via `fixture.editor.sidebar.designPanel.textStyle.fontFamily` or something similar.

Reply:
But we don't need font family in a fixture variable. I did this change because:
`openFontPicker` is used to click on the `Font family` button (so then you can test search etc), the issue occurs when we have multiple Font Pickers on the screen (Style tab + Floating Menu), most of the time we will have just one Font Picker on the screen and just in Floating Menu tests we will have two.